docs/docs/consumer.mdx

```diff
@@ -59,7 +59,7 @@ automatically and the following should suffice:
 ```python
 from zepben.evolve import connect_tls, SyncNetworkConsumerClient, Feeder
 
-with connect_tls(host="ewb.zepben.com", rpc_port=443) as channel:
+async with connect_tls(host="ewb.zepben.com", rpc_port=443) as channel:
```

Comment:
`grpc.aio.Channel` does not implement synchronous `__enter__` and `__exit__` AFAIK

Reply:
You should still be able to do:
```
channel = connect_tls(...)
```
Which makes more sense, as if you were using async you'd always want to use NetworkConsumerClient, rather than the Sync version.

I see you've also made this change in your other PR... maybe remove it from here and we'll fix it there?

Reply:
Reverted consumer.mdx.